src/main/resources/sql/postgre.sql

```diff
@@ -0,0 +1,52 @@
+-- Currently the Connection URL for the connector is jdbc:postgresql://localhost/ + database name (based on approach used)
+-- We should add a properties or config file to change it on the fly
```

Comment:
Adding a system property is a good idea. This should be addressed in a separate PR.